datafusion/physical-plan/src/joins/utils.rs

```diff
@@ -2069,20 +2077,25 @@ mod tests {
                 (10, Absent, Absent, Absent, Absent),
                 None,
             ),
-            // No min or max (or both).
+            // No min or max (or both), but distinct available.
             (
                 (10, Absent, Absent, Inexact(3), Absent),
                 (10, Absent, Absent, Inexact(3), Absent),
-                None,
+                Some(Inexact(33)),
```

Comment:
the behaviour of these two have changed because we will use the distinct values even though we have no range